test/unit/specs/pages/Autolending/LendTimingDropdown.spec.js

```diff
@@ -0,0 +1,177 @@
+			);
+		});
+		it('Previous auto lend 3', () => {
+			wrapper.setData({
+				enableAfter: 3,
+			});
+			expect(wrapper.find('.text-notice').text()).toBe(
+				"Notice: Your previous setting was 'If I don't log in for 3 Months', the closest matching setting above will be applied upon save." // eslint-disable-line max-len
+			);
+		});
+		it('Previous auto lend 6', () => {
+			wrapper.setData({
+				enableAfter: 6,
+			});
+			expect(wrapper.find('.text-notice').text()).toBe(
+				"Notice: Your previous setting was 'If I don't log in for 6 Months', the closest matching setting above will be applied upon save." // eslint-disable-line max-len
+			);
+		});
+		it('Previous auto lend 12', () => {
+			wrapper.setData({
+				enableAfter: 12,
+			});
+			expect(wrapper.find('.text-notice').text()).toBe(
+				"Notice: Your previous setting was 'If I don't log in for 1 Year', the closest matching setting above will be applied upon save." // eslint-disable-line max-len
+			);
+		});
+		it('Previous auto lend 18', () => {
+			wrapper.setData({
+				enableAfter: 18,
+			});
+			expect(wrapper.find('.text-notice').text()).toBe(
+				"Notice: Your previous setting was 'If I don't log in for 1.5 Years', the closest matching setting above will be applied upon save." // eslint-disable-line max-len
+			);
+		});
+		it('Previous auto lend other', () => {
+			wrapper.setData({
+				enableAfter: 32,
+			});
+			expect(wrapper.find('.text-notice').text()).toBe('');
+		});
+	});
+	describe('convertEnableAfterToNewSetting should convert old auto lend setting to new values', () => {
+		it('Previous auto lend 0 - Whenever I have credit available', () => {
+			expect(wrapper.vm.convertEnableAfterToNewSetting(0)).toBe(0);
+		});
+		it('Previous auto lend 1 - If I dont log in for 1 Month', () => {
+			expect(wrapper.vm.convertEnableAfterToNewSetting(1)).toBe(45);
+		});
+		it('Previous auto lend 2 - If I dont log in for 2 Months', () => {
+			expect(wrapper.vm.convertEnableAfterToNewSetting(2)).toBe(45);
+		});
+		it('Previous auto lend 3 - If I dont log in for 3 Months', () => {
+			expect(wrapper.vm.convertEnableAfterToNewSetting(3)).toBe(90);
+		});
+		it('Previous auto lend 6 - If I dont log in for 6 Months', () => {
+			expect(wrapper.vm.convertEnableAfterToNewSetting(6)).toBe(120);
+		});
+		it('Previous auto lend 12 - If I dont log in for 1 Year', () => {
+			expect(wrapper.vm.convertEnableAfterToNewSetting(12)).toBe(120);
+		});
+		it('Previous auto lend 18 - If I dont log in for 1.5 Years', () => {
+			expect(wrapper.vm.convertEnableAfterToNewSetting(18)).toBe(120);
+		});
+		it('Previous auto lend null or other value', () => {
+			expect(wrapper.vm.convertEnableAfterToNewSetting(1666)).toBe(0);
+			expect(wrapper.vm.convertEnableAfterToNewSetting(undefined)).toBe(0);
+			expect(wrapper.vm.convertEnableAfterToNewSetting(null)).toBe(0);
+		});
+	});
+	describe('Apollo mixin result callback should set the component variables correctly', () => {
```

Comment:
I think these might be better as cypress tests, where the api response to `{ my { autolendProfile } }` can be mocked with this data and then check that the correct option is currently selected in the dropdown.